provider/offline.go

```diff
@@ -842,6 +824,34 @@ func (r ResourceSchema) Validate() error {
 	return nil
 }
 
+func (r ResourceSchema) ToColumnStringSet(resType OfflineResourceType) (stringset.StringSet, error) {
+	set := make(stringset.StringSet)
+	switch resType {
+	case Feature:
+		set.Add(strings.ToUpper(r.Entity), strings.ToUpper(r.Value))
+		if r.TS != "" {
+			set.Add(strings.ToUpper(r.TS))
```

Comment:
This assumes that columns aren't case senstive

Reply:
Columns in Snowflake are returned in upper case, but I see you're point: we should also change anything returned from the offline store to uppercase, too, to ensure there aren't any case sensitivity issues.

Am I understanding your comment correctly?